Add details for neuron following (#2129)
dfinity/portal: docs/tokenomics/token-holders/nns-app-quickstart.md

- #### Step 4:  Verify the transaction information, then click **Confirm and Send**. 

## Neuron following

> To maximize neuron rewards and fully utilize the NNS's liquid democracy, neuron owners are encouraged to vote on every proposal submitted to the NNS. However, it may be hard for neuron owners to manually vote on every single proposal submitted, since several proposals may be submitted at one time, or neuron owners may not have the necessary level of technical understanding to evaluate proposals themselves. To aid in this, the NNS supports **neuron following**.

Reviewer: ```suggestion
To maximize neuron rewards and vote on every proposal submitted to the NNS, neuron holders can make use of the NNS's liquid democracy. It may be hard for neuron owners to manually vote on every single proposal submitted or neuron owners may not have the necessary level of technical understanding to evaluate proposals themselves. To aid in this, the NNS supports **neuron following**.
```

> A neuron can be configured to vote automatically by following the voting decision made by a group of other neurons. For example, if the neuron follows a group that votes to adopt a proposal with more than half of the followees voting in favorite of adoption, the neuron following that group will vote to approve the proposal as well. If the neuron follows a group that votes to reject a proposal with more than half of the followees voting to reject, the neuron following the group will vote to reject the proposal. 

Reviewer: Should we add a more explicit remark wrt the following:
I think many users thing following more neurons is better as it will ensure that a vote is cast for them. In many cases this is not true however: If a neuron follows 10 followees on a topic it will _only_ vote if 6 out of these 10 vote the same way. This means, that if 5 vote, then the following neuron will not cast a vote.
So if there are some reliable neurons, it can be the better option to actually follow less neurons.
Of course there can be other motivations to follow multiple experts - but I wonder if the above is something that some users don't understand well. Maybe we can also just not make any recommendations but mention an example e.g., with the 10 followees?

> Additionally, there is a catch-all follow rule that may be defined for a neuron to automatically vote on proposals for which no follow rule has been defined. It is assumed that neuron owners will follow other neurons that support the best interests of the network and their own economic interests.

Reviewer: ```suggestion
Additionally, there is a catch-all follow rule that may be defined for a neuron to automatically vote on proposals for which no follow rule has been defined. It is in each neuron's interest to follow other neurons that support the best interests of the network and their own economic interests.
```

> Additionally, there is a catch-all follow rule that may be defined for a neuron to automatically vote on proposals for which no follow rule has been defined. It is assumed that neuron owners will follow other neurons that support the best interests of the network and their own economic interests.

Reviewer: There are some exceptions for topics that are not included in the catch all. Should they be mentioned? 
They should be visible on the NNS FE dapp. I think they are SNS&Neurons'fund and Governance

> Additionally, there is a catch-all follow rule that may be defined for a neuron to automatically vote on proposals for which no follow rule has been defined. It is assumed that neuron owners will follow other neurons that support the best interests of the network and their own economic interests.

Reviewer: Maybe before saying "in addition there is a catch- all" we can say that following is done on the basis of topics? Or did I just miss this?

To configure your neuron to follow the group of another neuron:

- #### Step 1: From the NNS dapp, select 'My Neuron Staking', then select the neuron that you'd like to set a following rule for.

![NNS Neuron following 1](../_attachments/following1.png)

- #### Step 2: In the neuron's details, scroll down to the 'Following' section and select 'Follow Neurons'. 

![NNS Neuron following 2](../_attachments/following2.png)

- #### Step 3: Next, select a topic to follow. 

The topic refers to the different types of proposals; for example, the 'Governance' topic defines a following role for governance type proposals, while the 'Subnet Management' topic defines a following role for subnet management proposals. There is also a 'catch-all' rule that can be followed for all proposal types except for governance, SNS, and neurons' fund. 

For this example, select 'Governance'. 

![NNS Neuron following 3](../_attachments/following3.png)

- #### Step 4: Select 'Add Followee'.

![NNS Neuron following 4](../_attachments/following4.png)

- #### Step 5: Enter a Neuron ID to follow, or select a group of neurons from the menu provided. 

> In this example, select 'DFINITY Foundation'. 

Reviewer: We might want to consider making another example here to encourage people to try out something else and note how to get someon else's neuron ID . But this can also be an improvment for later.

A neuron can be configured to vote automatically by following the voting decision made by a group of other neurons. Following is done on the basis of topics, such that a neuron can choose to follow the voting decision for different proposal topics, such as governance or subnet management. For example, if the neuron has been configured to follow a group's decision for governance proposals, if the group votes to adopt a governance proposal with more than half of the followees voting in favor of adoption, the neuron following that group will vote to approve the proposal as well. If the neuron has been configured to follow a group's decision for subnet management proposals, if the group votes to reject a subnet management proposal with more than half of the followees voting to reject, the neuron following the group will vote to reject the proposal. 

:::caution

> It is important to note that when following a group of neurons, your neuron will only cast a vote if at least 6 out of 10 of the neurons within the followed group vote the same way. If only 5 neurons vote, your neuron will not cast a vote. Therefore, it is recommended to follow groups with at least 10 followees.

Reviewer: ```suggestion
It is important to note that when following a group of neurons, your neuron will only cast a vote if more than half of the neurons within the group vote the same way. For example, if you follow 10 neurons on a topic, your neuron only votes if at least 6 out of the 10 neurons vote the same way. If only 5 neurons vote, your neuron will not cast a vote. This should be considered when choosing to follow multiple neurons on a topic.
```

I think the example is great, but 10 was just a random example. I would refrase it slightly to make it more general and then add one concrete example. I think the conclusion sentence in the end was not right (or I did maybe not fuly follow it) therefore I also proposed an alternative sentence there

:::

Additionally, there is a catch-all follow rule that may be defined for a neuron to automatically vote on proposals for which no follow rule has been defined; however, this catch-all follow rule does not include SNS & neuron's fund or governance proposals. It is in each neuron's interest to follow other neurons that support the best interests of the network and their own economic interests.

To configure your neuron to follow the group of another neuron:

- #### Step 1: From the NNS dapp, select 'My Neuron Staking', then select the neuron that you'd like to set a following rule for.

![NNS Neuron following 1](../_attachments/following1.png)

- #### Step 2: In the neuron's details, scroll down to the 'Following' section and select 'Follow Neurons'. 

![NNS Neuron following 2](../_attachments/following2.png)

- #### Step 3: Next, select a topic to follow. 

> The topic refers to the different types of proposals; for example, the 'Governance' topic defines a following role for governance type proposals, while the 'Subnet Management' topic defines a following role for subnet management proposals. There is also a 'catch-all' rule that can be followed for all proposal types except for governance, SNS, and neurons' fund. 

Reviewer: ```suggestion
The topic refers to the different types of proposals; for example, the 'Governance' topic defines a following role for governanc-related proposals, while the 'Subnet Management' topic defines a following role for subnet management proposals. There is also a 'catch-all' rule that can be followed for all proposal types except for governance, SNS, and neurons' fund. 
```

(types is a fixed term we use for something else, therefore I would not use it here)
